packages/admin-rte/src/core/defaultBlocktypeMap.tsx

```diff
@@ -0,0 +1,129 @@
+        renderConfig: {
+            element: (p) => <Typography variant="h2" {...p} />,
+            aliasedElements: ["h2"],
+        },
+    },
+    "header-three": {
+        supportedBy: "header-three",
+        label: <FormattedMessage {...headerMessage} values={{ level: 3 }} />,
+        group: "dropdown",
+        renderConfig: {
+            element: (p) => <Typography variant="h3" {...p} />,
+            aliasedElements: ["h3"],
+        },
+    },
+    "header-four": {
+        supportedBy: "header-four",
+        label: <FormattedMessage {...headerMessage} values={{ level: 4 }} />,
+        group: "dropdown",
+        renderConfig: {
+            element: (p) => <Typography variant="h4" {...p} />,
+            aliasedElements: ["h4"],
+        },
+    },
+    "header-five": {
+        supportedBy: "header-five",
+        label: <FormattedMessage {...headerMessage} values={{ level: 5 }} />,
+        group: "dropdown",
+        renderConfig: {
+            element: (p) => <Typography variant="h5" {...p} />,
+            aliasedElements: ["h5"],
+        },
+    },
+    "header-six": {
+        supportedBy: "header-six",
+        label: <FormattedMessage {...headerMessage} values={{ level: 6 }} />,
+        group: "dropdown",
+        renderConfig: {
+            element: (p) => <Typography variant="h6" {...p} />,
+            aliasedElements: ["h6"],
+        },
+    },
+    blockquote: {
+        supportedBy: "blockquote",
+        label: <FormattedMessage id="cometAdmin.rte.controls.blockType.blockquote" defaultMessage="Blockquote" />,
+        renderConfig: {
+            // @TODO: A better default styling is needed
+            element: (p) => <Typography variant="overline" {...p} />, //@TODO: what should be default MUI-element here?
+            aliasedElements: ["blockquote"],
+        },
+    },
+    "unordered-list-item": {
+        supportedBy: "unordered-list",
+        group: "button",
+        label: <FormattedMessage id="cometAdmin.rte.controls.blockType.unorderedList" defaultMessage="Bulletpoints" />,
+        icon: FormatListBulletedIcon,
+        renderConfig: {
+            wrapper: <Typography component="ul" />,
+            element: "li", // Do not change this to a react component (<Typography {...p}/>) unless you implement the css for nesting lists yourself!
+            // This css is used by default and it handles the nesting: https://github.com/facebook/draft-js/blob/master/src/component/utils/DraftStyleDefault.css#L152
+            // But for this css to be applied "element" needs to be an (undecorated) li-element
+        },
+    },
+    "ordered-list-item": {
+        supportedBy: "ordered-list",
+        group: "button",
+        label: <FormattedMessage id="cometAdmin.rte.controls.blockType.orderedList" defaultMessage="Numbering" />,
+        icon: FormatListNumberedIcon,
+        renderConfig: {
+            wrapper: <Typography component="ol" style={{ marginLeft: 0, paddingLeft: 0 }} />,
```

Comment:
I don't think it's a good idea to set a default `paddingLeft`. If necessary, this should be done in the application. 
And there is no initial `marginLeft` on an `ol` tag so this should also not be set imo.

Also I don't think `ul` and `ol` should have different initial styling.

Reply:
> Also I don't think ul and ol should have different initial styling.

I agree, I change that.